frontend/src/features/projects/spl/forms/CloseOutFinancialSummaryForm.tsx

```diff
@@ -124,21 +123,21 @@ const CloseOutFinancialSummaryForm = (props: CloseOutFinancialSummaryFormProps)
         <Col md={1}>&nbsp;</Col>
         <Col>
           <ProjectNotes
-            field={`notes[${NoteTypes.Remediation}].note`}
```

Comment:
Using array positioning is an invalid implementation and will result in data corruption